wallets/index.js

```diff
@@ -218,34 +215,13 @@ export function useWallets () {
 
 export function useWallet (name) {
   const { wallets } = useWallets()
+  return wallets.find(w => w.def.name === name)
+}
 
-  const wallet = useMemo(() => {
-    if (name) {
-      return wallets.find(w => w.def.name === name)
-    }
-
-    // return the first enabled wallet that is available and can send
-    return wallets
-      .filter(w => !w.def.isAvailable || w.def.isAvailable())
-      .filter(w => w.config?.enabled && canSend(w))[0]
-  }, [wallets, name])
-
-  const { logger } = useWalletLogger(wallet?.def)
-
-  const sendPayment = useCallback(async (bolt11) => {
-    const decoded = bolt11Decode(bolt11)
-    logger.info(`↗ sending payment: ${formatSats(decoded.satoshis)}`, { bolt11 })
-    try {
-      const preimage = await wallet.def.sendPayment(bolt11, wallet.config, { logger })
-      logger.ok(`↗ payment sent: ${formatSats(decoded.satoshis)}`, { bolt11, preimage })
-    } catch (err) {
-      const message = err.message || err.toString?.()
-      logger.error(`payment failed: ${message}`, { bolt11 })
-      throw err
-    }
-  }, [wallet, logger])
-
-  if (!wallet) return null
-
-  return { ...wallet, sendPayment }
+export function useWalletsWithPayments () {
```

Comment:
mhh, I would call this `useSendWallets` since this does not return wallets with the `sendPayment` function

Reply:
oh good point